projects/GEDKeeper2/GKCore/LangMan.cs

```diff
@@ -1036,10 +1036,10 @@ public static class LangMan
             /* 238 */ "family",
             /* 239 */ "Namesakes",
             /* 240 */ "Remove the event?",
-            /* 241 */ "Detach link to note?",
-            /* 242 */ "Detach link to multimedia?",
-            /* 243 */ "Detach link to source?",
-            /* 244 */ "Remove the association?",
+            /* 241 */ "Are you sure you want to detach link to note?",
+            /* 242 */ "Are you sure you want to detach link to multimedia?",
+            /* 243 */ "Are you sure you want to detach link to source?",
+            /* 244 */ "Are you sure you want to remove the association?",

```

Comment:
To make changes to the default localization only once, you need to make changes only to LangMan. And then perform one run with the activated line SaveDefaultLanguage() in the constructor of the class MainWin. Then deactivate the line.